---
content_type: page
title: Labs
uid: c810141c-0282-3f29-da2a-83f1fe93dcb1
---

Contents:

*   [Lab Basics](#Basics)
*   [First Day: Lab Orientation](#Orientation)
*   [Module 1: RNA Engineering](#RNA_Engineering)
*   [Module 2: Protein Engineering](#Protein_Engineering)
*   [Module 3: Cell-Biomaterial Engineering](#Biomaterial_Engineering)

Lab work is divided into three modules of eight sessions each. Learning materials for each lab session (linked below under 'Lab Materials') include an introduction, experimental protocol, and a reagents list, followed by a "for next time" homework assignment. A short pre-lab lecture preceeds most of the lab sessions.

Selected results from some labs are included courtesy of the students and used with permission.

{{< anchor "Basics" >}}{{< /anchor >}}Lab Basics
------------------------------------------------

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
WORKING IN THE LAB
{{< thclose >}}
{{< thopen >}}
COMMUNICATING YOUR WORK
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}


[General lab policies, do's and don'ts]({{< baseurl >}}/pages/labs/general-lab-policies-do2019s-and-don2019ts)

[Guidelines for working in the tissue culture facility]({{< baseurl >}}/pages/labs/guidelines-for-working-in-the-tissue-culture-facility)

Guidelines on using personal protective equipment, developed by  
MIT's Environmental Health and Safety Office:

*   [Personal Protective Equipment](https://ehs.mit.edu/workplace-safety-program/personal-protective-equipment/)
*   Lab coat ([PDF](http://openwetware.org/images/4/47/2010_EHS_LabCoatGuidance.pdf))


{{< tdclose >}}
{{< tdopen >}}


[Statement on Collaboration and Integrity]({{< baseurl >}}/pages/syllabus/statement-on-collaboration-and-integrity)

[Guidelines for maintaining your lab notebook]({{< baseurl >}}/pages/labs/guidelines-for-maintaining-your-lab-notebook)

[Guidelines for oral presentations]({{< baseurl >}}/pages/assignments/guidelines-for-oral-presentations)

[Guidelines for writing up your research]({{< baseurl >}}/pages/assignments/guidelines-for-writing-up-your-research)


{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

{{< anchor "Orientation" >}}{{< /anchor >}}First Day: Lab Orientation
---------------------------------------------------------------------

There are ten stations for you and your lab partner to visit on your lab tour today. Some will be guided tours with a TA or faculty there to help you and others are self-guided, leaving you and your partner to try things on your own. Your visit to each station should last ~10-15 minutes. It doesn't matter which station you visit first but you must visit them all before you leave today. Afterward, you should have some time remaining to begin the For Next Time assignment, and are strongly encouraged to do so. Your lab practical next time will assess your mastery of a selection of these stations - all are fair game, as is the lab safety material covered in today's pre-lab lecture.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
PRE-LAB LECTURE
{{< thclose >}}
{{< thopen >}}
LAB Materials
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m0d1)) 
{{< tdclose >}}
{{< tdopen >}}
[Lab orientation]({{< baseurl >}}/pages/labs/20.109-s10-lab-orientation)
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

{{< anchor "RNA_Engineering" >}}{{< /anchor >}}Module 1: RNA Engineering
------------------------------------------------------------------------

**Instructors**: Jacquin Niles and Agi Stachowiak

In this module you will investigate RNA aptamer selection. You may already be familiar with peptides or proteins, such as antibodies, that bind to specific molecules. Short fragments of RNA – called RNA aptamers - can have secondary structures that also allow them to bind a target molecule with good affinity and specificity. Normally, RNA aptamers that bind particular targets are found by screening many candidates at random in a process called SELEX, or systematic evolution of ligands by exponential enrichment; however, predictive computational tools can also be used. In the coming weeks, you will essentially perform one round of SELEX. Because SELEX typically takes several rounds to isolate target-binding aptamers, you will start with a known aptamer mixture rather than than a completely random library. Your goal will be to explore what experimental parameters affect the enrichment of a heme-binding RNA aptamer from a mixture of heme-binding and non-binding RNAs.

**Acknowledgement**: We thank 20.109 instructor Natalie Kuldell for helpful discussions and for acquiring funding for module development.

{{< resource "d6cf9db1-840b-790f-a496-1a103d1291e4" >}}

Public domain image. (Prepared using RNA Folding (mfold) at the [mFold Web Server](http://mfold.rna.albany.edu/?q=mfold/download-mfold)). Reference: Zuker, M. "Mfold Web Server for Nucleic Acid Folding and Hybridization Prediction." _Nucleic Acids Res._ 31, no. 13 (2003): 3406-3415.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
{{< anchor "MODUle_1:_RNA_Engineering" >}}{{< /anchor >}}MODUle 1: RNA Engineering
{{< thclose >}}

{{< trclose >}}
{{< tropen >}}
{{< thopen >}}
LAB DAYS
{{< thclose >}}
{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}
{{< thopen >}}
LAB Materials
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 1
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d1))
{{< tdclose >}}
{{< tdopen >}}
[Amplify aptamer-encoding DNA]({{< baseurl >}}/pages/labs/module-1-day-1-amplify-aptamer-encoding-dna)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 2
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d2)) 
{{< tdclose >}}
{{< tdopen >}}
[Purify aptamer-encoding DNA]({{< baseurl >}}/pages/labs/module-1-day-2-purify-aptamer-encoding-dna)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 3
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d3)) 
{{< tdclose >}}
{{< tdopen >}}
[Prepare RNA by IVT]({{< baseurl >}}/pages/labs/module-1-day-3-prepare-rna-by-ivt)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 4
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d4))
{{< tdclose >}}
{{< tdopen >}}
[Purify RNA and run affinity column]({{< baseurl >}}/pages/labs/module-1-day-4-purify-rna-and-run-affinity-column)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 5
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d5)) 
{{< tdclose >}}
{{< tdopen >}}
[RNA to DNA by RT-PCR]({{< baseurl >}}/pages/labs/module-1-day-5-rna-to-dna-by-rt-pcr)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 6
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d6)) 
{{< tdclose >}}
{{< tdopen >}}
[Post-selection IVT and journal club]({{< baseurl >}}/pages/labs/module-1-day-6-post-selection-ivt-and-journal-club)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 7
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m1d7)) 
{{< tdclose >}}
{{< tdopen >}}
[Aptamer binding assay]({{< baseurl >}}/pages/labs/module-1-day-7-aptamer-binding-assay)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 8
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}
{{< tdopen >}}
[Journal club (contd.)]({{< baseurl >}}/pages/labs/module-1-day-8-journal-club-continued)
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

See the [Assignments]({{< baseurl >}}/pages/assignments) page for descriptions of the Module 1 laboratory report and RNA computational analysis assignments.

[TA notes for Module 1]({{< baseurl >}}/pages/labs/ta-notes-for-lab-module-1)

{{< anchor "Protein_Engineering" >}}{{< /anchor >}}Module 2: Protein Engineering
--------------------------------------------------------------------------------

**Instructors**: Alan Jasanoff and Agi Stachowiak

In this experiment, you will modify a protein called inverse pericam (developed by [Nagai, et al](http://www.ncbi.nlm.nih.gov/pubmed/11248055).) in order to affect its functions as a sensor. Inverse pericam (IPC) comprises a permuted fluorescent protein linked to a calcium sensor. The "inverse" in the name refers to the fact that this protein shines brightly in the absence of calcium, but dimly once calcium is added. The dissociation constant _KD_ of wild-type IPC with respect to calcium is reported to be 0.2 μM (see also figure below). Your goal will be to shift this titration curve or change its steepness by altering one of the calcium binding sites in IPC's calcium sensor portion. You will modify inverse pericam at the gene level using a process called site-directed mutagenesis, express the resultant protein in a bacterial host, and finally purify your mutant protein and assay its calcium-binding activity via fluorescence. In the course of this module, we will consider the benefits and drawbacks of different approaches to protein design, and the types of scientific investigations and applications enabled by fluorescently tagged biological molecules.

**Reference**: Nagai, T., et al. "[Circularly Permuted Green Fluorescent Proteins Engineered to Sense Ca2+](http://dx.doi.org/10.1073/pnas.051636098)." _PNAS_ 98, no. 6 (March 6, 2001): 3197-3202. \[Open Access\]

We gratefully acknowledge 20.109 instructor Natalie Kuldell for helpful discussions during the development of this module, as well as for her prior work in developing a [related module](http://openwetware.org/wiki/20.109:Module_2) in the Spring 2007 course.

{{< resource "c54f55f1-3f57-fa02-f1a1-6b41a696bf73" >}}

Raw titration curve for IPC. Shown here is sample data from the teaching lab: normalized fluorescence for wild-type inverse pericam as a function of calcium concentration. As you will later learn, an apparent KD can be estimated from such a plot: it is the point on the x-axis where the curve crosses y = 50%, or ~0.5 μM here

{{< resource "041afc0e-47f2-ad63-d7ff-f6d92b255a6d" >}}

Fitted titration curve for IPC. A more sophisticated analysis using curve-fitting reveals KD to be ~ 0.3 μM, closer to the reported value for inverse pericam.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
{{< anchor "Module_2:_Protein_Engineering" >}}{{< /anchor >}}Module 2: Protein Engineering
{{< thclose >}}

{{< trclose >}}
{{< tropen >}}
{{< thopen >}}
LAB DAYS
{{< thclose >}}
{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}
{{< thopen >}}
LAB Materials
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 1
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d1)) 
{{< tdclose >}}
{{< tdopen >}}
[Start-up protein engineering]({{< baseurl >}}/pages/labs/module-2-day-1-start-up-protein-engineering)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 2
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d2))
{{< tdclose >}}
{{< tdopen >}}
[Site-directed mutagenesis]({{< baseurl >}}/pages/labs/module-2-day-2-site-directed-mutagenesis)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 3
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d3))
{{< tdclose >}}
{{< tdopen >}}
[Bacterial amplification of DNA]({{< baseurl >}}/pages/labs/module-2-day-3-bacterial-amplification-of-dna)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 4
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d4))
{{< tdclose >}}
{{< tdopen >}}
[Prepare expression system]({{< baseurl >}}/pages/labs/module-2-day-4-prepare-expression-system)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 5
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d5))
{{< tdclose >}}
{{< tdopen >}}
[Induce protein and evaluate DNA]({{< baseurl >}}/pages/labs/module-2-day-5-induce-protein-and-evaluate-dna)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 6
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m2d6))
{{< tdclose >}}
{{< tdopen >}}
[Characterize protein expression]({{< baseurl >}}/pages/labs/module-2-day-6-characterize-protein-expression)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 7
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}
{{< tdopen >}}
[Assay protein behavior]({{< baseurl >}}/pages/labs/module-2-day-7-assay-protein-behavior)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 8
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}
{{< tdopen >}}
[Data analysis]({{< baseurl >}}/pages/labs/module-2-day-8-data-analysis)
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

See the [Assignments]({{< baseurl >}}/pages/assignments) page for a description of the Module 2 protein engineering research paper.

[TA notes for Module 2]({{< baseurl >}}/pages/labs/module-2-ta-notes)

{{< anchor "Biomaterial_Engineering" >}}{{< /anchor >}}Module 3: Cell-Biomaterial Engineering
---------------------------------------------------------------------------------------------

**Instructor**: Agi Stachowiak

_What makes a cell become one type and not another? How can we influence this process, and why would we even want to? When faced with conflicting information – in our own experiments, or in the broader scientific literature – how do we determine what is credible?_ These are just some of the questions you will explore in the third and final module, all in the context of tissue engineering. The goal of tissue engineering (also called regenerative medicine) is to repair tissues damaged by acute trauma or disease. Repair is stimulated by insertion of a porous scaffold at the wound or disease site; the scaffold may carry relevant mature or progenitor cells, and in some cases also soluble growth factors. In cartilage tissue, mature cells are called chondrocytes, and their progenitor cells are mesenchymal stem cells. Tissue regeneration shares many characteristics with natural tissue development, including the importance of appropriate cell differentiation and phenotype maintenance. You will perform a hypothesis-driven investigation of the effects of environmental manipulations on primary chondrocytes and/or mesenchymal stem cells. In particular, you will assess cell viability, genotype, and protein production, but the specific experimental question is up to you.

I gratefully acknowledge Professor Alan Grodzinsky and several members of his lab group (particularly Rachel Miller and Paul Kopesky), for their technical advice and stimulating discussions during the development of this module.

{{< resource "0201cdef-b5d0-bbcc-bcb5-e27b697090f4" >}}

Morphology of primary bovine chondrocytes grown under two different culture conditions. Optical micrographs of chondrocytes grown in monolayer (left) and alginate bead culture (right) are shown. Cells in the 3D culture retain a round phenotype, while cells on the flat surface extend processes and spread out.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
{{< anchor "Module_3:_Cell-Biomaterial_Engineering" >}}{{< /anchor >}}Module 3: Cell-Biomaterial Engineering
{{< thclose >}}

{{< trclose >}}
{{< tropen >}}
{{< thopen >}}
Lab DAYS
{{< thclose >}}
{{< thopen >}}
PRE-LAB LECTURES
{{< thclose >}}
{{< thopen >}}
LAB Materials
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 1
{{< tdclose >}}
{{< tdopen >}}
([PDF)]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d1)
{{< tdclose >}}
{{< tdopen >}}
[Start-up biomaterials engineering]({{< baseurl >}}/pages/labs/module-3-day-1-start-up-biomaterials-engineering)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 2
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}
{{< tdopen >}}
[Initiate cell culture]({{< baseurl >}}/pages/labs/module-3-day-2-initiate-cell-culture)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 3
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d3))
{{< tdclose >}}
{{< tdopen >}}
[Testing cell viability]({{< baseurl >}}/pages/labs/module-3-day-3-testing-cell-viability)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 4
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d4))
{{< tdclose >}}
{{< tdopen >}}
[Preparing cells for analysis]({{< baseurl >}}/pages/labs/module-3-day-4-preparing-cells-for-analysis)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 5
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d4))
{{< tdclose >}}
{{< tdopen >}}
[Transcript-level analysis]({{< baseurl >}}/pages/labs/module-3-day-5-transcript-level-analysis)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 6
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d6))
{{< tdclose >}}
{{< tdopen >}}
[Protein-level analysis]({{< baseurl >}}/pages/labs/module-3-day-6-protein-level-analysis)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 7
{{< tdclose >}}
{{< tdopen >}}
([PDF]({{< baseurl >}}/resources/mit20_109s10_labpre_m3d7))
{{< tdclose >}}
{{< tdopen >}}
[Wrap-up analysis]({{< baseurl >}}/pages/labs/module-2-day-7-assay-protein-behavior)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Day 8
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}
{{< tdopen >}}
Student presentations
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

See the [Assignments]({{< baseurl >}}/pages/assignments) page for a description of the Module 3 cell-biomaterial engineering report.

[TA notes for Module 3]({{< baseurl >}}/pages/labs/20.109-s10-ta-notes-for-module-3)